cypress/integration/mood-page-test.js

```diff
@@ -3,7 +3,12 @@ describe('Mood page', () => {
     cy.visit('http://localhost:3000/mood')
   });
 
-  it('Should have a Kanye face', () => {
+  it('Should have a cook Kanye album-art inspired background image', () => {
```

Comment:
Just curious what a 'cook Kanye album-art inspired background' is? Is the 'cook' just a typo or am I missing something?